WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -432,6 +433,23 @@ private extension NewOrderViewModel {
             .assign(to: &$navigationTrailingItem)
     }
 
+    /// Updates the notice based on the `orderSynchronizer` sync state.
+    ///
+    func configureSyncErrors() {
+        orderSynchronizer.statePublisher
+            .sink { [weak self] state in
+                guard let self = self else { return }
+                switch state {
+                case .error(let error):
+                    self.notice = NoticeFactory.syncOrderErrorNotice(with: self.orderSynchronizer)
+                    DDLogError("⛔️ Error syncing new order remotely: \(error)")
+                default:
+                    self.notice = nil
+                }
+            }
+            .store(in: &self.cancellables)
+    }
```

Comment:
Super nit: if you want this to read more "reactive" you could map the `statePublisher` into a `notice` and then use the `assign` operator.

And then you could log the error in the `handleEvents` operator.

The final output is the same, but you can differentiate easily between the reactive chain's main purpose(convert the state to notice) and the side effects handling (logging).